src/App.tsx

```diff
@@ -1,56 +1,92 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person>();
+  const [inputFocus, setInputFocus] = useState(false);
```

Comment:
```suggestion
  const [isInputFocused, setIsInputFocused] = useState(false);
```
![image](https://github.com/mate-academy/react_autocomplete/assets/96409563/f9384860-cb2d-43b9-b475-d6a107aea865)